lib/inspect_task.rb

```diff
@@ -79,6 +79,10 @@ def build_description(store, name, system, scopes, filter, options)
     scopes.each do |scope|
       inspector = Inspector.for(scope).new(system, description)
       Machinery::Ui.puts "Inspecting #{Machinery::Ui.internal_scope_list_to_string(inspector.scope)}..."
+      if scope == "patterns" && description.packages &&
+          description.packages.package_system == "dpkg"
```

Comment:
I think this should go to the patterns inspector instead. That also has the benefit that you already know whether it's a `dpkg` based system or not, so you don't have to rely on the packages scope to be around.